### DISCUSSION TITLE
Start at Windows Startup & Not syncing account

### DISCUSSION THREAD

**Discussion by Trak-X** [bkaankose/Wino-Mail#403] [Q&A]
Hi. I have came across the app on Reddit and wanted to give it a try but i encountered 2 issues and im trying to fix them:

1. Cannot enable the app to start at launch, even if i already set it as "Let this app run in background always" from Settings. When i click the enable button i get the message in the image and nothing else happens
![image](https://github.com/user-attachments/assets/25f80815-57dc-4e0c-a1b1-1d09109c9f5c)

2. I tried the limit, 3 accounts. All 3 of them Google accounts, 2 are loading properly, one its just empty / no sync. I did notice at some point, if i remove permission from My Account section in Google Account, the app does get a trigger for missing permission, but no matter how much i leave it open after adding the account, it does not load anything. Is there anything i can check locally to fix this?

**Comment by bkaankose** [comment]
1. You can enable Wino for startup on Task Manager as well. Go to Task Manager > Startup Apps > Right click to Wino Mail and enable it. I don't really know why you had this message on enabled case to be honest.
2. I don't think it's a permission issue. You can delete the account that doesn't work and re-add. If it still doesn't work, you can share the logs from Settings -> About -> Diagnostics with me via either on Discord or mail to bkaankose {at} outlook {dot} com. I can take a look.

**Comment by Trak-X** [comment]
Following your answer, i tried debugging. 

The first issue was resolved by resetting the Group Policy in Windows 11. Somehow it was not allowing the app to set the autostart rule, and even if i was doing it from Task Manager, it was putting it on Disabled on its own.

For the second issue, i left the computer connected and powered on for 4-5 hours with the account connected .. and after about 2 hours it started showing the emails (tho no loading bar as for the other accounts was present)

Can you tell me or add it as a improvement request, is there already a way to see attachments in received emails? I got 2 emails today for example with attachments but i could only see them in the browser, in Chrome, not in the email window. (i am versed in Kotlin and Java but i can try to do something in C# too)

Thanks.

**Comment by bkaankose** [comment]
v1 had was using separate process for synchronization called Wino Server.

in v2 with WinUI3 migration, this is now deprecated and everything is baked into a single process. Therefore, this discussion is not relevant anymore. Background sync process should not have any issues with group policies or etc.

Closing as fixed.
